proto/conjunction.proto

```diff
@@ -0,0 +1,203 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+syntax = "proto3";
+
+import "proto/answer.proto";
+import "proto/concept.proto";
+
+package typedb.protocol;
+
+message Conjunction {
```

Comment:
So since the scope of this right now is just within "Analyze", and it's not a generic "Conjunction" really, let's settle on "AnalyzedConjunction" and call the file `analyzed_conjunction.proto` , what do you think?